src/fenicsxconcrete/names.py

```diff
@@ -0,0 +1,26 @@
+from pydantic.dataclasses import dataclass
+from enum import Enum
+from typing import Literal
+
+FieldVaraibles = Literal["displacement", "temperature", "nonlocal_equivalent_strain"]
+
+QuadratureVariables = Literal["mandel_stress", "mandel_strain", "mandel_strain_rate"]
+
+@dataclass
+class Quantity:
+    unit: str
+
+class BaseQuantity(Enum):
+    Displacement = Quantity("m")
+    Velocity = Quantity("m/s")
+    Acceleration = Quantity("m/s^2")
+    Temperature = Quantity("K")
+    Strain = Quantity("1")
+    Stress = Quantity("Pa")
+    Density = Quantity("kg/m^3")
+    Time = Quantity("s")
+
```

Comment:
How would I use that if I have other, additional quantities? Would I create my own class AnnikaQuantity(Enum) or do we add that here? Some question for the SolutionFields Class ...